src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -74,6 +75,7 @@ case class BenchmarkResult(
     planningTime: Option[Double] = None,
     executionTime: Option[Double] = None,
     result: Option[Long] = None,
+    doubleResults: Map[String, Double] = Map.empty,
```

Comment:
Can you describe more what you are doing here?  Once we add things to this schema we are stuck with them forever.